### ISSUE TITLE
The train set is smaller than the test set

### ISSUE THREAD

**Issue by hebecked** [UniversalDependencies/UD_Livvi-KKPP#2]
Is there any specific reason that the train set is smaller than the test set?
Also, there is no dev set.

Would anybody mind if I switch them out and take out some samples from the (new) train set to create a dev set?

**Comment by jnivre** [comment]
Yes. The prescribed strategy for growing a UD treebank (since the shared task in 2017) is to first develop a test set, to guarantee that it is possible to perform evaluation, for example, in zero-shot learning experiments. Once a test set exists, the next step is to start growing the training set, which initially will be smaller than the test set but over time will grow much larger. Once enough training data has been accumulated, a dedicated dev set may be created. Before that, techniques such as cross-validation on the training set can be used for tuning experiments.

It is crucial that the test set remains the same and is only used for final evaluation. If we start changing splits, this will inevitably lead to train-test leakage, because people may not be aware that their test data has been included in the training set by someone else. So please respect the integrity of the test set and do not ever use any part of it for training.

**Comment by jnivre** [comment]
Most importantly, there may already be published results based on this test set. If the test set changes, this will make replication harder and may result in misleading comparisons despite everyone's best intentions.

**Comment by dan-zeman** [comment]
See also [the guidelines for data splits](https://universaldependencies.org/release_checklist.html#data-split).
